editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -5231,8 +5232,17 @@ CanvasItemEditor::CanvasItemEditor(EditorNode *p_editor) {
 	controls_vb = memnew(VBoxContainer);
 	controls_vb->set_begin(Point2(5, 5));
 
+	HBoxContainer *controls_hb = memnew(HBoxContainer);
+	controls_vb->add_child(controls_hb);
+
+	button_center_view = memnew(Button);
+	controls_hb->add_child(button_center_view);
+	button_center_view->connect("pressed", callable_mp(this, &CanvasItemEditor::_popup_callback), make_binds(VIEW_CENTER_TO_SELECTION));
+	button_center_view->set_flat(true);
+	button_center_view->set_tooltip(TTR("Center View"));
```

Comment:
I think we usually set up the node before adding it to the tree and connecting signals. But that's only a nit.

Reply:
Right, the signals are always connected last. But after/before adding is not really consistent. Node setup is done after adding to tree in many places.